web/webkit/src/test/scala/net/liftweb/webapptest/ToHeadUsages.scala

```diff
@@ -167,7 +167,7 @@ object ToHeadUsages extends Specification  {
     }
 
     "Exclude from context rewriting" in {
-      val first = http.Req.fixHtml("/wombat",
+      val first = http.Req.normalizeHtml("/wombat",
```

Comment:
Per [the relevant commit](https://github.com/lift/framework/pull/1841/commits/002fe6846401e4d0c6fbdaafa24329e1ff5e9909):

> Fix a couple of webkit test deprecation warnings.
>
> These are related to the renaming of Req.fixHtml.

I renamed `fixHtml` to `normalizeHtml` while doing the event extraction work, to bring consistency to the concept of normalizing HTML throughout the system.